v22.1/manage-certs-vault.md

```diff
@@ -573,11 +573,11 @@ function install_roach_on_node {
     gcloud compute ssh $1 --command 'if [[ ! -e cockroach-v21.2.4.linux-amd64 ]];
```

Comment:
Question: can this be converted to something generic as well? This would make sense as-is if, say, this procedural only applies to clients running exactly v21.2.4, but I don't find any copy on page that agrees with this read.

Reply:
@MichaelTrestman am I clear to set this v22.1 instance to dynamic also, or must it pin to exactly `v21.2.4`? Thanks!

Reply:
a dynamic pin to the current version is perfect.